### DISCUSSION TITLE
Enabling TLS

### DISCUSSION THREAD

**Discussion by KravTeam** [symfony/symfony#45744] [Q&A]
Hello,
I have enabled TLS via symfony
> `symfony server:ca:install`
Now if I want to connect in http I have to use commande symfony:
> `symfony serve --no-tls`   // it's working
otherwise I am in https when I do:
> `symfony serve`               // it's working

but then, if if use:
> `php -S localhost:8000 -t public -d display_errors=1`     // it's not working !
I get server issues like "404 not found" for some ressources:
127.0.0.1:56174 [404]: POST /admin/club/get/profil/image/1622eb22499576.jpg - No such file or directory

Can some one explain me ?
1622eb22499576.jpg is not in public directory is there a connection or is it because of TLS enabled ?

**Comment by chyshyya** [comment]
Is `/admin/club/get/profil/image/1622eb22499576.jpg` exists at all?

**Reply by chyshyya** [reply]
And why POST?

**Reply by KravTeam** [reply]
Yes 1622eb22499576.jpg exist and the POST request is working when I do `symfony serve --no-tls` (http) or `symfony serve` (https) but not when I do `php -S localhost:8000 -t public -d display_errors=1`.

**Reply by KravTeam** [reply]
Thank you Chyshyya for your answer. Ihave done `symfony server:ca:uninstall` to disable TLS. It's working like before.
However, I still have a weird behavior using php build in server different from `symfony serve` command.
I'll certainly ask the question in an anther post.

**Reply by KravTeam** [reply]
I found the reason of the 404 error, it is beacause I had to activate ACL on my server I think it imay not be possible with `php -S localhost:8000 -t public ` command.

https://symfony.com/doc/current/setup/file_permissions.html#1-using-acl-on-a-system-that-supports-setfacl-linux-bsd
